src/index.html

```diff
@@ -5,8 +5,359 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__wrap">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__wrap">
          <li class="nav__item">
            <a class="nav__link is-active" href="#Apple">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#Samsung">Samsung</a>
          </li>
```
add blanks between multiline elements